install/kubernetes/helm/istio/values-istio-multicluster-gateways.yaml

```diff
@@ -2,7 +2,6 @@ global:
   # Provides dns resolution for global services
   podDNSSearchNamespaces:
   - global
-  - "[[ valueOrDefault .DeploymentMeta.Namespace \"default\" ]].global"  
```

Comment:
this change needs a revert after the proper selection of DNSDomains is done.

Reply:
what do you mean? you have to add the namespace to list of searched namespaces right?

Reply:
Did you mean `$(POD_NAMESPACE).svc.cluster.local`?

Reply:
corrected